report_bundler/bundler.py

```
@@ -0,0 +1,73 @@
+import csv
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)